ui/src/pages/backoffice/Document/DocumentDetails/DocumentDetails.js

```diff
@@ -38,7 +38,7 @@ export default class DocumentDetails extends Component {
     const samePidFromRouter =
       prevProps.match.params.documentPid === documentPid;
     if (!samePidFromRouter) {
-      this.props.fetchDocumentsDetails(documentPid);
+      this.props.fetchDocumentDetails(documentPid);
```

Comment:
Since in the constructor there is `this.fetchDocumentDetails = this.props.fetchDocumentDetails` why not use `this.fetchDocumentDetails` instead of `this.props.fetchDocumentDetails`?

Reply:
Nicely spotted thanks!
I have removed it from constructor, I think that @topless was proposing to just use `props` and avoid to set it in the constructor.

Reply:
I suggested that in order to avoid extra constructors for assigning props, and to prevent people (myself included) from renaming stuff, `fetchDocumentsDetails` would become `getDocument` and so on.